src/omnibelt/parallel-limit-map-p.js

```diff
@@ -0,0 +1,64 @@
+const curry = require('ramda/src/curry');
+const times = require('ramda/src/times');
+
+const arrayEvaluator = async (func, accumulator, iterator) => {
+  const next = iterator.next();
+  if (next.done) { return; }
+  accumulator.push(await func(next.value));
+  await arrayEvaluator(func, accumulator, iterator);
+};
+
+const objectEvaluator = async (func, accumulator, iterator) => {
+  const next = iterator.next();
+  if (next.done) { return; }
+  const key = next.value[0];
+  accumulator[key] = await func(next.value);
```

Comment:
ℹ️ Good stuff! I was trying to work out a way these functions could be combined but couldn't quite get there. Doesn't really matter, but if they could be, it would reduce this code and the branching in the primary function.

```js
// non-functioning code

const evaluator = setter => async (func, accumulator, iterator) => {
  const next = iterator.next();
  if (next.done) { return; }
  await arrayEvaluator(func, setter(accumulator, func, next.value), iterator);
};


const arrayEvaluator = evaluator((/* Needs too much context */) => {
  return accumulator.push(await func(next.value));
});
```

Reply:
I made a pass at this. I had attempted it a bit when moving the code into here, but decided it wasn't worth it - you made me give it another go. Probably not exactly what you were thinking, but close.

Reply:
👍